src/hardware/sblaster.cpp

```diff
@@ -599,25 +630,23 @@ static void DMA_Silent_Event(Bitu val) {
 	if (sb.dma.left) {
 		Bitu bigger=(sb.dma.left > sb.dma.min) ? sb.dma.min : sb.dma.left;
 		float delay=(bigger*1000.0f)/sb.dma.rate;
-		PIC_AddEvent(DMA_Silent_Event,delay,bigger);
+		PIC_AddEvent(DMA_Suppress_Samples, delay,bigger);
 	}
 }
 
-static void END_DMA_Event(Bitu val) {
-	GenerateDMASound(val);
-}
-
-static void CheckDMAEnd(void) {
+static void DMA_Flush_Remaining() {
```

Comment:
Unless there's a really good reason, I would advise against using `DMA_` prefix for static functions, as it suggests these are public functions declared in `include/dma.h` and implemented in `src/hardware/dma.cpp`. (Same comment to all new static functions or static functions with new names in this file).

*That being said, this code was not consitent before your change, but maybe let's not make it even less consistent? :)*

Reply:
> Unless there's a really good reason

These four `DMA_*` functions specifically operate on the sound blaster DMA-transfers instead of DAC transfers. Will trying to think of something better :).